test/formulas/importFormulas.test.js

```diff
@@ -24,8 +24,35 @@ describe('importFormulas', () => {
       processId: 2,
     });
   });
+  it('should be able to handle for empty file or directory', async () => {
+    let formulasData = await readFormulasFromDir(formulasDirectoryPath);
+    expect(formulasData).not.toBeNull();
+    formulasData = mapIndex(
```

Comment:
`formulasData` is not being used anywhere. Is this testing existence of some value or not needed?

Reply:
It's more like a flow that we perform on any operation/task and each test case represents some sort of different flows. You might see read data in some directory in each test case as it gets executed every time we call doctor-core.